docker/release/Dockerfile

```diff
@@ -0,0 +1,29 @@
+FROM cita/cita-run:ubuntu-18.04-20190829
+LABEL maintainer="Cryptape Technologies <contact@cryptape.com>"
+
+## Install CITA release here
+ARG ENCRYPTION_ALG=secp256k1
+ARG HASH_ALG=sha3
+RUN mkdir -p /opt/cita
+
+WORKDIR /opt/cita
+COPY cita_${ENCRYPTION_ALG}_${HASH_ALG}.tar.gz .
+
+# Keep tar package here for md5 test
+RUN tar -xf cita_${ENCRYPTION_ALG}_${HASH_ALG}.tar.gz \
+    && mv cita_${ENCRYPTION_ALG}_${HASH_ALG}/* /opt/cita \
+    && rm -rf cita_${ENCRYPTION_ALG}_${HASH_ALG}
+
+# Set CITA_HOME
+ENV CITA_HOME=/opt/cita
+# Set PATH to cita binary
+ENV PATH=/opt/cita/bin:${PATH}
+## End of install CITA release
+
+COPY entrypoint.sh /usr/bin/
+RUN chmod +x /usr/bin/entrypoint.sh
+
+WORKDIR /opt/cita-run
+
+ENTRYPOINT ["/usr/bin/entrypoint.sh"]
+CMD ["cita", "bebop", "help"]
```

Comment:
I seriously thought about this automatic detection mechanism. I think it can cover most OS and docker version, but not every. So I will keep the sub-command `bebop` for cita.sh, and use this sub-command for `Dockerfile`.